### PR TITLE
Mastermunj main - Added comments on ToWords class methods

### PR DESCRIPTION
**Pull request by apoorvachikara**

### CODE REVIEW

**Comment by apoorvachikara** [comment]
I have made changes - 
1) Added comments on the functions in the main file
2) Added the recent dependency merge in package-lock.json

File: dist/ToWords.d.ts [hunk 1]
```diff
@@ -5,9 +5,33 @@ export declare class ToWords {
...
     constructor(options?: ToWordsOptions);
+    /**
+     * getting language locale class based on user
+     * passed config options
+     * It contains the mapping for currency, texts and
+     * numberToWord mapping
+     *
+     * @returns {class} - based on selected currency
+     */
     getLocaleClass(): ConstructorOf<LocaleInterface>;
```
Comment (mastermunj) [inline dist/ToWords.d.ts:16]: @apoorvachikara Files inside dist gets deleted on each build. Best is to have code comments in the source files.

File: package-lock.json [hunk 2]
```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "to-words",
```
Comment (mastermunj) [inline package-lock.json:8]: Please help with more details on changes in the package-lock as it's unclear.

**Review by mastermunj — CHANGES_REQUESTED** [review]

File: dist/ToWords.d.ts (hunk 1 above)
Comment (apoorvachikara) [reply dist/ToWords.d.ts:16]: I put them in the ts file only, the action might create a new dist with comments on it.

File: package-lock.json (hunk 2 above)
Comment (apoorvachikara) [reply package-lock.json:8]: I took the update from the main branch for vulnerabilities - the pull updated versions for certain modules

Comment (mastermunj) [reply package-lock.json:8]: I'll suggest you to remove package-lock from your PR. I periodically take care of maintaining the package updates.

Comment (apoorvachikara) [reply package-lock.json]: ok, let me remove and update the PR.

**Comment by apoorvachikara** [comment]
I have removed the package-lock.json. You can check now. The conflicts are resolved.

**Pull request closed without merging**